indico/modules/rb_new/controllers/backend/bookings.py

```diff
@@ -134,3 +136,20 @@ class RHRoomSuggestions(RHRoomBookingBase):
     @use_args(search_room_args)
     def _process(self, args):
         return jsonify(get_suggestions(args, limit=NUM_SUGGESTIONS))
+
+
+class RHBookingDetails(RHRoomBookingBase):
+    def _process_args(self):
+        self.booking = Reservation.get_one(request.view_args['booking_id'])
+
+    def _process(self):
+        attributes = reservation_details_schema.dump(self.booking).data
+        date_range, occurrences = get_booking_occurrences(self.booking)
+        date_range = [dt.isoformat() for dt in date_range]
+        occurrences = {dt.isoformat(): reservation_details_occurrences_schema.dump(data).data
+                       for dt, data in occurrences.iteritems()}
+        edit_logs = sorted(reservation_edit_log_schema.dump(self.booking.edit_logs).data,
```

Comment:
you can do the sorting on the schema as well, see the changes in schemas.py in #3582 for an example